### ISSUE TITLE
Handle DEVp2p handshake and other messages

### ISSUE THREAD

**Issue by germsvel** [poanetwork/mana#166]
Currently, our ExWire tcp connection is handling the encrypted handshake. 

We now need to be able to handle a DEVp2p handshake (Hello messages) as well as Disconnect, Ping, and Pong messages. All of these messages should only be send/received if the crypto handshake was successful. 

For more on DEVp2p protocol, see https://github.com/ethereum/wiki/wiki/%C3%90%CE%9EVp2p-Wire-Protocol

Note that all messages must be framed. For more on framing see https://github.com/ethereum/devp2p/blob/master/rlpx.md#framing.

**Comment by hayesgm** [comment]
Was this closed by #410?

**Comment by InoMurko** [comment]
Can we close this @hayesgm ?

**Comment by germsvel** [comment]
Yeah, I think #410 may have handled this. I'm not sure how robust it is, but it seems like this can be closed.
